main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Dog
+//  action 1: The dog barks.
+dog.bark();
+//  action 2: The dog sleeps.
+dog.sleep();
+//  action 3: The dog eats.
+dog.eat();
 
 //  2)
+//  Noun: Radio
+//  action 1: The radio receives signals.
+radio.receive();
+//  action 2: The radio transmits signals.
+radio.transmit();
+//  action 3: The radio changes frequencies.
+radio.setFrequency();
+
+//  3) musicPlayer
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  action 1: The music player plays a song.
+musicPlayer.play();
+//  action 2: The music player stops playing a song.
+musicPlayer.stop();
+//  action 3: The music player opens a new file.
+musicPlayer.openFile();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Car 
+//  action 1: The car accelerates.
+car.accelerate();
+//  action 2: The car slows down.
+car.break();
+//  action 3: The car changes gear. 
+car.changeGear();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Bird
+//  action 1: The bird flys.
+bird.fly();
+//  action 2: The bird sings.
+bird.sing();
+//  action 3: The bird lays an egg.
+bird.layEgg();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Student
+//  action 1: The student reads.
+student.read();
+//  action 2: The student takes notes.
+student.takeNotes();
+//  action 3: The student takes exam.
+student.takeExam();
+
+//  7) 
+//  Noun: Teacher
+//  action 1: The teacher lectures.
+teacher.lecture();
+//  action 2: The teacher grades assignments.
+teacher.gradeAssignment();
+//  action 3: The teacher takes attendance.
+teacher.takeAttendance(); 

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.